frontend/src/components/TaskMapEdge.tsx

```diff
@@ -1,11 +1,52 @@
-import { FC } from 'react';
+import { FC, MouseEvent, useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import classNames from 'classnames';
 import { getMarkerEnd, getBezierPath } from 'react-flow-renderer';
+import { StoreDispatchType } from '../redux';
+import { roadmapsActions } from '../redux/roadmaps';
 import css from './TaskMapEdge.module.scss';
 
 const classes = classNames.bind(css);
 
 const DrawPath: FC<any> = ({ id, d, markerEnd }) => {
+  const [selected, setSelected] = useState<string | undefined>(undefined);
+  const dispatch = useDispatch<StoreDispatchType>();
+
+  // Should be given 'id' as param in the form: 'from-taskId-to-taskId-in-roadmapId'
+  const deleteRelation = async (tbdeleted: string) => {
+    const data = tbdeleted.split('-');
+    await dispatch(
+      roadmapsActions.removeTaskRelation({
+        from: Number(data[1]),
+        to: Number(data[3]),
+        type: 0,
+        roadmapId: Number(data[5]),
+      }),
+    );
+    dispatch(roadmapsActions.getRoadmaps());
+  };
+
+  const handleDelete = (e: MouseEvent) => {
+    // Needs to have this to prevent opening context-menu
+    e.preventDefault();
+    deleteRelation(id);
+  };
+
+  useEffect(() => {
+    const handleBackspace = (e: any) => {
+      // 8 = backspace, 46 = delete-button
+      if (selected && (e.keyCode === 8 || e.keyCode === 46)) {
+        deleteRelation(selected);
+        setSelected(undefined);
+      }
+    };
+    document.addEventListener('keydown', handleBackspace);
+    return () => {
+      document.removeEventListener('keydown', handleBackspace);
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [deleteRelation, dispatch]);
```

Comment:
Should this include the `deleteRelation` as dependency? It could be wrapped in a `useCallback`